### ISSUE TITLE
Private Key Re-Use Detected. All my transactions are confirmed

### ISSUE THREAD

**Issue by dwest2** [iotaledger/wallet#894]
Hello,  all my transactions are confirmed yet I still get the Re-Use Detected error. Please advise
RBAEZ9LQMOQIKTVZNYBZKMDVKOWQWKBMH9XSLTJAWULDZQYXLRCAABBTTRISVCVVSLBZJBLYOVVYNXHACZHNGUTFXD
Thanks

**Comment by b-f-** [comment]
Same here. Here's how to repeat:
- spend from address, the receive (2.5.4)
- make a list of all transactions in wallet history by hand
- reattach/promote (2.5.6) all the 0 amount pending transactions until every unique address says either "Persistence: Confirmed" or "Persistence: A reattachment confirmed"
- send 1 iota, success 
NOTE: wallet balance is not updated!
- send the rest of amount -> "private key reuse detected" again

RESULT: amount was taken from wrong address! My other addresses with amount > 0 has unchanged balance. 

Proof:
https://iotasear.ch/hash/YYGKICECYMXNOB9LBIFJBJDUERQITZIOW9RGTIFKDJQMNEBEGZJOLFTYSMJFYTNRMQYLYYXCXHRJKEFYADHPUUBMMA
This is the address with all the balance on this seed:
https://iotasear.ch/hash/FYZQZINSGCVA9LZBICUDMXDBJEVFVITL99TQYLDHKBRLEBL9CVVCMAITHPL9FPVWHXGQVFKYKJ9BGBEJYZTIEBVXFW

I tried the following to resolve the issue (not working):
- update the above list of all transactions in wallet history and check status of every single one again (all unique addresses are "Persistence: Confirmed" or "Persistence: A reattachment confirmed")
- generate new receive address, wait until confirmed (reattach/promote), repeat

**Comment by wonkytonky** [comment]
![image](https://user-images.githubusercontent.com/19390834/34407931-3d6f353c-ebc1-11e7-8c1b-08ed56bd15f2.png)

looks like you send money to  used address.  hence the private key reuse detect error .

**Comment by wonkytonky** [comment]
@dwest2  what wallet did you use, what operating system etc.; give more info for developers . seems remaining coins didn't move to new address like wallet normaly does

**Comment by dwest2** [comment]
So you were unable to make it work? What is your next step? Thanks

**Comment by onemoreitguy** [comment]
I'm on the same boat as you @b-f- 
When you make a payment the address is "disabled" but after that payment I got a receive confirmation.
Now I cannot transfer that one anywhere, because a payment/reuse was already done :(

**Comment by rajivshah3** [comment]
Please don't post issues/help requests like this here, instead post on the [forum](http://forum.helloiota.com). Thanks!

**Comment by dwest2** [comment]
I had the same issue for a week trying everything and ' ERROR: PRIVATE KEY REUSE DETECTED! would come up. A fellow named Richard contacted me and said to send him my private seed and he would send me the coins. I was like, how crazy is this, of course I am not sending my key. I asked him how to fix it and for 3 days he emailed me directions, YouTube videos, links, zip files, program downloads etc. I installed everything needed on the third day and then asked what the next step is. He said to set up a proxy server to attach to the tangle and then use 'power shell' commands to unlock the stuck IOTA. I tried setting up the Proxy server but for some reason, I could not connect to the Tangle. I sort of gained trust in him so I said I will send him the private key and hope for the best. 10 minutes later I received my 500 IOTA and he charged me only 2%. I have his email but I would have to ask if I can publish it here. He seems to be a really amazing programmer and a fine fellow. I hope he can help you, think he is in Austria. Donald
